backend/de.metas.business/src/main/java/de/metas/banking/api/BankRepository.java

```diff
@@ -124,8 +136,25 @@ public Bank createBank(@NonNull final BankCreateRequest request)
 		// ESR:
 		record.setESR_PostBank(request.isEsrPostBank());
 
+		record.setC_DataImport_ID(DataImportConfigId.toRepoId(request.getDataImportConfigId()));
+
 		saveRecord(record);
 
 		return toBank(record);
 	}
+
+	public DataImportConfigId retrieveDataImportConfigIdForBank(@Nullable final BankId bankId)
+	{
+		if (bankId == null)
+		{
+			return retrieveDefaultBankDataImportConfigId();
+		}
+
+		final Bank bank = getById(bankId);
+
+		final DataImportConfigId dataImportConfigId = bank.getDataImportConfigId();
+
+		return dataImportConfigId != null ? dataImportConfigId : retrieveDefaultBankDataImportConfigId();
```

Comment:
- here you can use CoalesceUtils.coalesceSupplierNotNull afais
- also AFAIU you should annotate the method with `@NonNull` to help intellij helping us